index.html

```diff
@@ -12968,6 +12968,44 @@ <h2>A project for first-time contributions</h2>
             <!-- ============== TEMPLATE ============== -->
             <!-- COPY everything BELOW this, from contributor card start to end along with the "START" and "END" comment lines -->
 
+            <!-- ________ Contributor's card START ________  -->
+            <div class="card">
+                <p class="name">Siddharth</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="https://twitter.com/Siddharthnpm" target="_blank">@Siddharthnpm
+                </p>
+                <p class="about">
+                    I am a B.tech student from India, I have recently started to explore this field of Website Development and I am liking it already! :).
+                    I am a huge blues fan and often find myself with the audio-only company of Buddy Guy, Keith Richards and Mr. Muddy Waters.
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="https://twitter.com/search?q=%23100DaysOfCode&src=typeahead_click" target="_blank" title="Introduction to the Dev community on Twitter">
+                                #100DaysOfCode
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://medium.com/javascript-in-plain-english" target="_blank"
+                                title="Really helpful article">
+                                Javascript in plain English by Medium.com
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.coursera.org/learn/html-css-javascript-for-web-developers" target="_blank" title="Really useful Course for developers">
+                                HTML, CSS, and Javascript for Web Developers on Coursera
+                            </a>
+                        </li>
+                    </ul>     <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+       
+                </div>
+            </div>
+            <!-- ________ Contributor's card END ________  -->
+
+            <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
+            
             <!-- ________ Contributor's card START ________  -->
```

Comment:
The template card for a contributor has been modified, please make sure it is restored back to older form which looks like this:
After this change please make a commit.
`            <!-- ============== TEMPLATE ============== -->
            <!-- COPY everything BELOW this, from contributor card start to end along with the "START" and "END" comment lines -->

            <!-- ________ Contributor's card START ________  -->
            <div class="card">
                <p class="name">Name</p>
                <p class="contact">
                    <i class="fab fa-twitter"></i>
                    <a href="YOUR twitter account link" target="_blank">@yourHandle</a>
                </p>
                <p class="about">
                    Write here one sentence about yourself. Anything you want: Why do you program, a fun fact, why
                    do you like programming, what you hope to achieve, your pet's name...
                </p>
                <div class="resources">
                    <p>3 Useful Dev Resources</p>
                    <ul>
                        <li>
                            <a href="#" target="_blank" title="This is a link to an awesome resource">
                                Link to resources
                            </a>
                        </li>
                        <li>
                            <a href="#" target="_blank"
                                title="This is another link to web development tool I use a lot">
                                Link to resources
                            </a>
                        </li>
                        <li>
                            <a href="#" target="_blank" title="The third link to something useful for developers">
                                Link to resources
                            </a>
                        </li>
                    </ul> 
       
                </div>
            </div>
            <!-- ________ Contributor's card END ________  -->`